client/rest/src/db/CatapultDb.js

```diff
@@ -204,11 +204,13 @@ class CatapultDb {
 	 * Retrieves filtered and paginated blocks.
 	 * @param {Uint8Array} signerPublicKey Filters by signer public key
 	 * @param {Uint8Array} beneficiaryAddress Filters by beneficiary address
+	 * @param {uint64} fromTimestamp Filters by fromTimestamp
+	 * @param {uint64} toTimestamp Filters by toTimestamp
```

Comment:
try something a little more descriptive, e.g.

Filters blocks by only including blocks with a timestamp greater than or equal to provided value
Filters blocks by only including blocks with a timestamp less than or equal to provided value

Reply:
I used your text verbatim 🙏